test_nand.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from nand import Nand
+
+FILE_PATH = "ssd_nand.txt"
+LBA_LENGTH = 100
+DEFAULT_VALUE = "0x00000000"
+FIRST_ADDRESS = 0
+LAST_ADDRESS = LBA_LENGTH - 1
+
+@pytest.fixture
+def nand():
+    return Nand(LBA_LENGTH, FILE_PATH)
+
+def test_nand_read_uninitialized_data_at_start_address(nand):
+    assert nand.read(FIRST_ADDRESS) == DEFAULT_VALUE
+
+def test_nand_read_uninitialized_data_at_last_address(nand):
+    assert nand.read(LAST_ADDRESS) == DEFAULT_VALUE
+
+def test_nand_write_first_address(nand):
+    addr = FIRST_ADDRESS
+    wdata = "0x1234ABCD"
```

Comment:
혹시 아래와 같이 소문자 테스트는 NAND가 아니라 SSD쪽에서 테스트 추가가 될 예정인지요?
```suggestion
    wdata = "0x1234abcd"
```